### PR TITLE
Paella7 fix trimming url params

### PR DESCRIPTION
**Pull request by miesgre**
### Your pull request should…

* [x] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by mliradelc** [comment]
Have you tried how it behaves with Wowza stream security?

**Comment by miesgre** [comment]
> Have you tried how it behaves with Wowza stream security?

No. We don't have Wowza stream security. But I assume it will work properly. Since all requests are made in the same way as with the normal video (without soft-trimming).

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Paella Player 7 - URL Parameters
+==============================
+
+Parameter      | Example    | Description
+---------------|------------|------------
+**id**         | `SOME-ID`  | Video Id to play
+**time**       | `10m20s`   | Seeks initially automatically to a specified time
+**trimming**   | `1m2s:10m` | Apply a soft trimming to the video
```
Comment (ziegenberg) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: The "possible values" in line 39 use a semicolon `;` but here you use a colon `:` as separator.

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Default value: `-`    
+        
+
+trimming
+--------
+Apply a soft trimming to the video.
+The result is that the user will only be able to view the specified video chunk. Useful to put links to parts of the video.
+
+Possible values
+* `<start time>;<end time>`
```
Comment (ziegenberg) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md:40]: The separator should also be explicitly mentioned and specified.

**Review by ziegenberg — CHANGES_REQUESTED** [review]

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md (hunk 1 above)
Comment (mliradelc) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: The **log** parameter and the description is missing.

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+Paella Player 7 - URL Parameters
+==============================
+
```
Comment (mliradelc) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: Add an introduction

```suggestion
The player support different URL parameters to modify the behaviour or the target event.
```

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    * `XhYmZs`
+* Minutes (with value `Y`) and seconds (with value `Z`)
+    * `YmZs`
+* Minutes (with value `Y`) only
+    * `Ym`
+* Seconds (with value `Z`) only
+    * `Zs`
+
+Default value: `-`
+
```
Comment (mliradelc) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md:55]:
```suggestion

Security Parameters
--------------------

If Stream Security is enabled, there will be additional parameters. For further information, kindly refer to the [stream security section](Link-to-stream-security).
```

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md (hunk 1 above)
Comment (miesgre) [reply docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: Fixed!

**Review by mliradelc — COMMENTED** [review]
For me, it's superb. 

As you said, this should not interfere with the stream security parameters, but we need to test it before merge. Currently, I don't have a `14.x` cluster to try it with our Wowza instance.

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md (hunk 1 above)
Comment (miesgre) [reply docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: Done!

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+Security Parameters
+--------------------
+
+If Stream Security is enabled, there will be additional parameters. For further information, kindly refer to the [stream security section](Link-to-stream-security).
```
Comment (mliradelc) [inline docs/guides/admin/docs/modules/paella.player7/url.parameter.md]:
```suggestion
If Stream Security is enabled, there will be additional parameters. For further information, kindly refer to the [stream security section](#modules/stream-security/).
```
I don't know if that link work, but should be like that, or not @gregorydlogan?

**Review by mliradelc — CHANGES_REQUESTED** [review]
Only a little change needed

File: docs/guides/admin/docs/modules/paella.player7/url.parameter.md (hunk 5 above)
Comment (miesgre) [reply docs/guides/admin/docs/modules/paella.player7/url.parameter.md]: Fixed!

**Review by mliradelc — APPROVED** [review]

**Pull request merged**